src/dhttest/cases/neo/Mirror.d

```diff
@@ -549,9 +548,8 @@ public class MirrorRemoveChannel : NeoDhtTestCase
                 {
                     case started:
                         // As soon as the mirror is up, remove the channel
-                        this.dht.assign(
-                            this.dht.removeChannel(this.test_channel,
-                                &delNotifier));
+                        this.dht.neo.removeChannel(this.test_channel,
+                            &delNotifier);
```

Comment:
Fun fact: this can be `null` ;-) (not that I would recommend that, I think this way it's clearer).